features-json/css-nesting.json

```
@@ -658,9 +658,9 @@
       "22":"n",
       "23":"a #3",
       "24":"a #3",
-      "25":"a #3",
-      "26":"a #3",
-      "27":"a #3"
+      "25":"n",
+      "26":"n",
+      "27":"n"
```

`n` would mean no support at all, you probably indeed `y` for full support? :)

ref: https://github.com/Fyrd/caniuse/blob/main/CONTRIBUTING.md#supported-changes ("stats" bullet point)

```suggestion
      "25":"y",
      "26":"y",
      "27":"y"
```